packages/@aws-cdk/custom-resource-handlers/lib/custom-resources-framework/runtime.ts

```diff
@@ -0,0 +1,133 @@
+export enum RuntimeFamily {
+  NODEJS,
+  PYTHON,
+  JAVA,
+  RUBY,
+}
+
+/**
+ * Properties used to initialize a framework runtime.
+ */
+interface RuntimeProps {
+  /**
+   * Whether or not this runtime is deprecated.
+   *
+   * @default false
+   */
+  readonly isDeprecated?: boolean;
+}
+
+/**
+ * Custom resource framework runtimes used for code generation.
+ */
+export class Runtime {
```

Comment:
Oof. This looks like a straight up copy and paste from the lambda module. That's a bit risky. Why do we need this?